k8s/0011.dd-cleaner-cronjob-1.yml

```
@@ -0,0 +1,32 @@
+                memory: 200Mi         # Reserve 200Mi RAM
+            env:                       # Environment variables passed into the container
+              - name: DATADOG_CREDENTIALS
+                valueFrom:
+                  secretKeyRef:
+                    name: datadog
+                    key: CREDENTIALS
+              - name: DATADOG_BASE_URL
+                value: "https://api.datadoghq.com/api/"  # Datadog API base URL
+          restartPolicy: Never         # Do not restart completed or failed jobs          
+          imagePullSecrets:
+            - name: regcred            # Credentials for pulling from private registry          
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)